### PR TITLE
mesos create

### PR DESCRIPTION
**Pull request by ranjithamca**

### CODE REVIEW

File: templates/ubuntu/tpl_mesosslave_install.go [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+func (tpl *UbuntuMesosSlaveInstall) Options(t *templates.Template) {
+
+	if masterip, ok := t.Options[MasterIp]; ok {
+		tpl.masterip = masterip
+}
+if sparkversion, ok := t.Options[SparkVersion]; ok {
+  tpl.sparkversion = sparkversion
+}
+fmt.Println(tpl.sparkversion)
```
Comment (thomasalrin) [inline templates/ubuntu/tpl_mesosslave_install.go]: @ranjitha remove this line

**Comment by indykish** [comment]
@morpheyesh  Verify and merge  it.

File: templates/ubuntu/tpl_mesosmaster_install.go [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+ Shell("export SPARK_HOME='/var/lib/meglytics/spark-"+Sparkvrs+"'"),
+)
+
+  pkg.AddCommands("sh",
+    WriteFile(sparkhome+"spark-jobserver/config/megam.sh", fmt.Sprintf(MEGAMSH, ip, "root", "root", sshkey,  installdir, Sparkvrs, job, executeuri), "root", 0755),
+)
+pkg.AddCommands("run",
+Shell("cd "+sparkhome+"spark-jobserver/bin;./server_deploy.sh megam"),
+)
+pkg.AddCommands("cadvisor",
```
Comment (morpheyesh) [inline templates/ubuntu/tpl_mesosmaster_install.go]: @ranjithamca Remove the cadvisor stop

**Comment by indykish** [comment]
@ranjithamca  Is this fixed from your end ?

**Pull request merged**